jupyterhub/log.py

```diff
@@ -162,7 +164,14 @@ def log_request(handler):
         location='',
     )
     msg = "{status} {method} {uri}{location} ({user}@{ip}) {request_time:.2f}ms"
-    if status >= 500 and status != 502:
```

Comment:
I think we could just add `and status not in {502, 503}` for this. The reason 502 is here is really the same as what you are after.

Reply:
I saw that and it looked *so* tempting to filter out 503, but didn't know if 503 had some other meaning.  If you approve I'll just replace this whole diff with that.

It looks like the two uses of 503 are related to stopped servers, so I'll go ahead and do this.  thanks!

Reply:
We log this extra stuff for debugging purposes when the error is likely the fault on the Hub side. 502 and 503 are errors that are generally "not my fault," so don't warrant the extra debugging.

It's possible that we should only be doing it for 500 exactly, which is the case for unhandled server-side errors